attachment_base_synchronize/data/cron.xml

```diff
@@ -8,8 +8,9 @@
             <field name="numbercall">-1</field>
             <field name="active">False</field>
             <field name="doall" eval="False" />
-            <field name="model">ir.attachment.metadata</field>
-            <field name="function">run_attachment_metadata_scheduler</field>
+            <field name="model_id" ref="model_ir_attachment_metadata"/>
+            <field name="state">code</field>
+            <field name="code">model.run_attachment_metadata_scheduler()</field>
```

Comment:
Shouldn't this be `model_id.run_....` ?

Reply:
Nope, it's model.cron() [example](https://github.com/OCA/OCB/blob/11.0/addons/account_asset/data/account_asset_data.xml#L7)